compiler/natives/src/net/http/cookiejar/example_test.go

```diff
@@ -0,0 +1,5 @@
+// +build js
+
+package cookiejar
+
+func ExampleNew() {}
```

Comment:
It's very helpful (for later) to put a comment inside that explains why a test/example is disabled:

```Go
func ExampleNew() {
    // Example relies on httptest.NewServer, which GopherJS does not implement.
}
```